index.js

```diff
@@ -1,18 +1,42 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "blue"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const person = document.querySelector("#profile")
+const pic = person.querySelector("img")
+const h2Name = person.querySelector("h2")
+const h2Nick = person.querySelector("em")
 
+pic.src = traveler.photo
+pic.alt = traveler.name
+
+h2Name.innerText = traveler.name
+h2Nick.innerText = traveler.nickname
 
 /***** Deliverable 4 *****/
 
+traveler.animalSightings.forEach(function(animal){
+    // console.log(animal)
+    const div = document.createElement("div")
+    div.innerHTML = `<li data-id="${animal.id}">
+                        <p>${animal.description}</p>
+                        <img src="${animal.photo}" alt="${animal.species}"/>
+                        <a href="${animal.link}" target="_blank">Here's a video about the ${animal.species} species!</a>
+                    </li>`
+    const collectionUl = document.querySelector('ul')
+    collectionUl.append(div)
+})
```

Comment:
This is on the correct track and will work fine except the formatting would not be ideal.  Instead of creating 'div', you should create 'li'.  HTML tags have built-in formatting. Putting an li inside of a ul will indent the content and make it easier to read. creating a div for each instance will add extra spacing around each animal sighting card.  Not the end of the world but you can do without it.  Once you create the li, you can do 
li.dataset.id = animal.id
and then 
li.innerHTML =  `<p>${animal.description}</p>
                        <img src="${animal.photo}" alt="${animal.species}"/>
                        <a href="${animal.link}" target="_blank">Here's a video about the ${animal.species} species!</a>`

Only other note is that the collectionUl variable can be saved above the function since it is not changing and this will save you an extra operation for each instance in animalSightings